src/frontends/pytorch/src/transforms/rfftn_complex_replacer.hpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "openvino/pass/pass.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace pass {
+
+class RFFTNComplexReplacer : public ov::pass::MatcherPass {
```

Comment:
Please add some description for the transformation

Reply:
Comments were added